Sources/Sentry/SentryHttpTransport.m

```diff
@@ -76,8 +80,16 @@ - (void)    sendEvent:(SentryEvent *)event
 // TODO: needs refactoring
 - (void)sendEnvelope:(SentryEnvelope *)envelope
    withCompletionHandler:(_Nullable SentryRequestFinished)completionHandler {
-    // TODO: Apply type based rate limiting
-    if (![self isReadyToSend:@""]) {
+    
+    if (![self.options.enabled boolValue]) {
```

Comment:
m: why do we check this field?

> Is the client enabled?. Default is @YES, if set @NO sending of events will be prevented.

Client calls the Transport so it seems not logical

Reply:
It was already there. I'm not 100% sure why, I just kept it to not break anything. Maybe we can remove it in the future.

Reply:
I see, got it, its because it appears as a new line here, its merged anyway, thx